helm-chart/templates/cvat_kvrocks/service.yml

```diff
@@ -15,8 +15,8 @@ spec:
     {{- include "cvat.labels" . | nindent 4 }}
   type: ClusterIP
   ports:
-  - port: 6666
-    targetPort: 6666
+  - name: tcp-kvrocks
```

Comment:
Maybe you could use `redis` instead? It would more accurately identify the protocol.